entries/s/scotiabank.com.json

```diff
@@ -1,13 +1,18 @@
 {
-  "Scotiabank": {
+  "Scotiabank Canada": {
     "domain": "scotiabank.com",
-    "url": "https://www.scotiabank.com",
```

Comment:
I've tried debugging the error and there seems to be a problem with receiving a 302 to an HTTP location when the original requested URI is an HTTPS site.

Here's the debug output:
```
= Request

! CONNECT TO scotiabank.com:443
! CONNECTION ESTABLISHED
GET / HTTP/1.1
User-Agent: 2FactorAuth/URLValidator (HTTPClient/2.8.3 on Ruby/2.7.2; +https://2fa.directory/bot)
From: https://2fa.directory/
Accept: */*
Date: Wed, 29 Sep 2021 19:52:20 GMT
Host: scotiabank.com



= Response

HTTP/1.1 302 Moved Temporarily
Server: AkamaiGHost
Content-Length: 0
Location: http://www.scotiabank.com/
Date: Wed, 29 Sep 2021 19:52:20 GMT
Connection: keep-alive

Traceback (most recent call last):
        8: from ./tests/validate-urls.rb:30:in `<main>'
        7: from ./tests/validate-urls.rb:30:in `each'
        6: from ./tests/validate-urls.rb:36:in `block in <main>'
        5: from ./tests/validate-urls.rb:21:in `check_url'
        4: from /var/lib/gems/2.7.0/gems/httpclient-2.8.3/lib/httpclient.rb:743:in `get'
        3: from /var/lib/gems/2.7.0/gems/httpclient-2.8.3/lib/httpclient.rb:854:in `request'
        2: from /var/lib/gems/2.7.0/gems/httpclient-2.8.3/lib/httpclient.rb:1111:in `follow_redirect'
        1: from /var/lib/gems/2.7.0/gems/httpclient-2.8.3/lib/httpclient.rb:1111:in `call'
/var/lib/gems/2.7.0/gems/httpclient-2.8.3/lib/httpclient.rb:730:in `default_redirect_uri_callback': redirecting to non-https resource (HTTPClient::BadResponseError)
```